client/tests/webdriver/baseSpecs/showOrganization.spec.js

```diff
@@ -142,4 +142,12 @@ describe("Show organization page", () => {
       )
     })
   })
+
+  describe("When on the show page of an organization with entity avatar", () => {
+    it("We should see the avatar", async() => {
+      await ShowOrganization.open(ORGANIZATION_UUID)
+      await (await ShowOrganization.getEntityAvatar()).waitForExist()
+      await (await ShowOrganization.getEntityAvatar()).waitForDisplayed()
```

Comment:
Where is the expect?

Reply:
That is implicit: if the UI element doesn't appear within the timeout, the test will fail.